src/main/java/org/example/model/Order.java

```diff
@@ -0,0 +1,48 @@
+package org.example.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "orders")
+@NoArgsConstructor
+@Getter
+public class Order {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @ManyToOne
+  @JoinColumn(name = "customer_id", nullable = false)
+  private Customer customer;
+
+  private String status;
+  private String paymentMethod;
```

Comment:
Instead of `paymentMethod`, use `totalPrice` here, which should be the sum of the `price` * `quantity` of all `LineItem`s associated to this order. This should update every time we are updating or adding a `LineItem`.